packages/plugin-firestore/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.3.22",
   "sideEffects": false,
   "description": "Magnetar plugin firestore",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/types/index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
```

Comment:
@egoist should we go for `.cjs` for commonJS and `.js` for esm? as a way of forward thinking